catalog/pipeline/fbc/0.1/README.md

```diff
@@ -0,0 +1,13 @@
+# Milestone-9 FBC Pipeline
```

Comment:
let's remove any mention of Milestones here.

Reply:
+1